fuzz-testing/src/main/scala/org/apache/comet/fuzz/Meta.scala

```diff
@@ -106,4 +106,8 @@ object Meta {
     Function("stddev_samp", 1),
     Function("corr", 2))
 
+  val unaryArithmeticOps: Seq[String] = Seq("+", "-")
+
+  val binaryArithmeticOps: Seq[String] = Seq("+", "-", "*", "/", "%", "&", "|", "^")
```

Comment:
any plans to support`<, <=, ==, <=>, >, >=, =, !` ?

Reply:
Thsi PR was focussing on arithmetic because it is related to the ongoing work to support ANSI mode, but yet, we should add tests for comparison operators as well